build-logic/common-plugins/src/main/kotlin/DebugBuild.kt

```diff
@@ -97,24 +97,22 @@ fun Project.printResolvedConfigFiles(level: LogLevel = LIFECYCLE, configName: St
     val files = config.resolvedConfiguration.resolvedArtifacts.map { it.file }
     logger.log(level, "$prefix Configuration '$configName' has ${files.size} files:")
     files.forEach {
-        logger.log(level, "$prefix    file: '$it'")
+        logger.log(level, "$prefix file: '$it'")
     }
```

Comment:
This whitespace was deliberately there, since there is a lot of information, and it is significantly simpler to read the output where we have a lot of resolved artifacts, if we have a title line with the name of the config and the number of files, followed by each of these files on a separate line, but indented.

Reply:
I'm using the "--info" quite often now and the amount of printout is overwhelming. I can only analyze it with some additional tools (less, grep, etc.) Didn't realize that that particular extra space was important, looked like a random omission.